sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs

```diff
@@ -66,7 +66,7 @@ public struct BlobSasBuilder : IEquatable<BlobSasBuilder>
         /// When specifying a range of IP addresses, note that the range is
         /// inclusive.
         /// </summary>
-        public IPRange IPRange { get; set; }
+        public SasIPRange SasIPRange { get; set; }
```

Comment:
Let's just call this `public SasIPRange IPRange { get; set; }`.  The Sas on the property name would be a little much since they're already using a `*SasBuilder`.

Reply:
Same for Queues and Files too.  I'd literally just F2 on the property names though.  No need to change any internal constants, etc.